workflow/usage.rst

```diff
@@ -175,12 +175,63 @@ what actions are allowed on a blog post::
     $transitions = $workflow->getEnabledTransitions($post);
 
 Using Events
-------------
+============
 
 To make your workflows even more powerful you could construct the ``Workflow``
-object with an ``EventDispatcher``. You can now create event listeners to
-block transitions (i.e. depending on the data in the blog post). The following
-events are dispatched:
+object with an ``EventDispatcher``. If you are using the ``Workflow`` component within
+the Symfony standard edition, this is already done for you.
+
+When a workflow subject's state is updated, the component dispatches several events that
+enable you to alter the component's behaviour:
+
+* ``workflow.[workflow name].guard.[transition name]``: dispatched before applying a transition.
+* ``workflow.[workflow name].leave.[state name]``: a subject leaves a state.
+* ``workflow.[workflow name].transition.[transition name]``: a transition is being applied.
+* ``workflow.[workflow name].enter.[state name]``: a subject enters a state.
+* ``workflow.[workflow name].announce.[transition name]``: a transition was applied to the subject.
```

Comment:
Perhaps this deserves a note that a `GuardEvent` will be dispatched for guards instead of a base `Workflow\Event`.